.github/workflows/deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Build and deploy
 
 on:
   push:
-    branches: ['master']
+    branches: ['development']
```

Comment:
According to your requirement it should be `develop`. or you want to make the branch name `development` ?

Reply:
I apologize. It is going to be `development`.